geonode/maps/templates/leaflet_maps/map_embed_widget.html

```diff
@@ -0,0 +1,68 @@
+<div class="map-container">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.0.3/leaflet.css"/>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.0.3/leaflet.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/proj4js/2.3.15/proj4-src.js"></script>
+    <script type="text/javascript">
+        $(document).ready(function () {
+            var wmsLayer = null;
+            var firstProjection = proj4('EPSG:3857');
+            var secondProjection = proj4('EPSG:4326');
+
+            var point_wsg84 = proj4(firstProjection, secondProjection, [{{ resource.center_x }}, {{ resource.center_y }}]);
```

Comment:
the resource has its own srid in `resource.srid`. You should use this to set the projection. instead of assuming it as EPSG:4326 (use 4326 if it is empty).